lib/msal-core/src/telemetry/ApiEvent.ts

```diff
@@ -0,0 +1,92 @@
+import TelemetryEvent from "./TelemetryEvent";
+import { EVENT_NAME_PREFIX, TELEMETRY_BLOB_EVENT_NAMES } from "./TelemetryConstants";
+import { scrubTenantFromUri, hashPersonalIdentifier } from "./TelemetryUtils";
+import { Logger } from "../Logger";
+
+// export these to test... could namespace them, or find another way?
+export const AUTHORITY_KEY = EVENT_NAME_PREFIX + "authority";
+export const AUTHORITY_TYPE_KEY = EVENT_NAME_PREFIX + "authority_type";
+export const PROMPT_KEY = EVENT_NAME_PREFIX + "ui_behavior";
+export const TENANT_ID_KEY = EVENT_NAME_PREFIX + "tenant_id";
+export const USER_ID_KEY = EVENT_NAME_PREFIX + "user_id";
+export const WAS_SUCESSFUL_KEY = EVENT_NAME_PREFIX + "was_successful";
+export const API_ERROR_CODE_KEY = EVENT_NAME_PREFIX + "api_error_code";
+export const LOGIN_HINT_KEY = EVENT_NAME_PREFIX + "login_hint";
```

Comment:
Since you can't use an enum with a computer value, I think an object would be better:

```js
export const EVENT_KEYS = {
   AUTHORITY: `${EVENT_NAME_PREFIX}authority`,
  ...
};
```

Reply:
fixing